docs/src/install/docker.md

```diff
@@ -0,0 +1,117 @@
+# With Docker
+
+This documentation section explains how to install lockc on a single machine
+with Docker. In order to do that, we need to install `lockcd` binary and a
+systemd unit for it.
```

Comment:
Side note: is there a lockcd rpm?

Reply:
Not yet, but I'm thinking about it. I would wait for #49 though - removing libbpf and LLVM dependency will make packaging much easier. I already started working on aya rewrite, I'm hoping to have some PR next week.